pyi.py

```diff
@@ -1008,6 +1008,29 @@ def run(self, tree: ast.AST) -> Iterable[Error]:
         yield from self.errors
 
 
+_TYPE_COMMENT_REGEX = re.compile(r"# type: (?!ignore)([^#]+)( ?#.*?)?$")
```

Comment:
Should we allow other whitespace? This also came up in Black (https://github.com/psf/black/pull/2698) but I haven't had a chance yet to look into what type checkers support.

Reply:
[Revealed type appears to be `int` for all of these](https://mypy-play.net/?mypy=latest&python=3.10&flags=no-strict-optional&gist=ac7f85aeaf85923e9e471a25cb9b7321) if `--no-strict-optional` is applied, so mypy certainly supports quite a few ways of doing type comments:

```python
a = None #type: int
b = None #  type: int
c = None #   type:   int
d = None#    type: int
e = None#type:int
```